context/app/static/js/components/Detail/ProvTabs/ProvTabs.jsx

```diff
@@ -6,18 +6,27 @@ import { StyledTab, StyledTabs, StyledTabPanel } from './style';
 import ProvGraph from '../ProvGraph';
 import ProvTable from '../ProvTable';
 import ProvAnalysisDetails from '../ProvAnalysisDetails';
+import { checkDataTypesForValues, getTabIndex } from './utils';
 
 function ProvTabs(props) {
   const { uuid, assayMetadata, provData } = props;
-  const { metadata, entity_type, ancestors } = assayMetadata;
+  const { metadata, entity_type, ancestors, data_types } = assayMetadata;
 
   const [open, setOpen] = React.useState(0);
   const handleChange = (event, newValue) => {
     setOpen(newValue);
   };
 
+  const shouldDisplayTable = !(
+    entity_type === 'Dataset' &&
+    data_types &&
+    checkDataTypesForValues(data_types, ['snare_lab', 'TMT-LC-MS'])
+  );
```

Comment:
```suggestion
  const shouldDisplayTable = !checkDataTypesForValues(data_types, ['snare_lab', 'TMT-LC-MS']);
```
Handle null data_types inside the function? Don't need to explicitly check entity type?